### PR TITLE
doc: Fix a small typo in README.md

### PR DESCRIPTION
**Pull request by rz-vastdata**

### CODE REVIEW

**Review by viirya — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -125,7 +125,7 @@ Comet shuffle feature is disabled by default. To enable it, please add related c
 ```
 
 Above configs enable Comet native shuffle which only supports hash partiting and single partition.
```
Comment (viirya) [inline README.md:127]:
```suggestion
Above configs enable Comet native shuffle which only supports hash partition and single partition.
```

Comment (viirya) [reply README.md:127]: Found another typo.

Comment (rz-vastdata) [reply README.md]: Thanks! Fixed in d3c6623.

**Review by andygrove — APPROVED** [review]
Thanks @rz-vastdata

**Comment by viirya** [comment]
Thanks @rz-vastdata

**Pull request merged**